library/src/main/kotlin/com/liferay/vulcan/consumer/screens/ThingScreenlet.kt

```diff
@@ -60,9 +60,9 @@ class ThingScreenlet(context: Context, attrs: AttributeSet) : BaseScreenlet(cont
     }
 
     init {
-        val typedArray = context.theme.obtainStyledAttributes(attrs, R.styleable.ThingScreenlet, 0, 0)
+        val typedArray = attrs?.let { context.theme.obtainStyledAttributes(it, R.styleable.ThingScreenlet, 0, 0) }
```

Comment:
I mean, there are 4 constructors in a view and I'm afraid we have to implement them all, right now we're just doing one